Source/Controllers/DataImport/SPDataImport.m

```diff
@@ -183,7 +183,7 @@ - (void)importFromClipboard
 	if([[[NSPasteboard generalPasteboard] stringForType:NSStringPboardType] length] > 4000)
 		[importFromClipboardTextView setString:[[[[NSPasteboard generalPasteboard] stringForType:NSStringPboardType] substringToIndex:4000] stringByAppendingString:@"\n…"]];
 	else
-		[importFromClipboardTextView setString:[[NSPasteboard generalPasteboard] stringForType:NSStringPboardType]];
+		if([[NSPasteboard generalPasteboard] stringForType:NSStringPboardType] != nil)[importFromClipboardTextView setString:[[NSPasteboard generalPasteboard] stringForType:NSStringPboardType]];
```

Comment:
Prefer this was a if() {} full block with brackets for clarity

Reply:
done